src/scenes/Login/LogInSignUp.js

```diff
@@ -0,0 +1,42 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import 'bootstrap/dist/css/bootstrap.min.css'
+import { BrowserRouter as Router, Switch, Route, Link } from 'react-router-dom'
+import Login from '../Login/components/Login'
+import SignUp from './components/SignUp'
+
+const LoginSignUp = () => {
+  return (<Router>
+    <div className="App">
```

Comment:
why you move the App class to here?

Reply:
when i start server, so, i need one UI contain css the same App.js . Maybe false, i will change it!